tests/rbd/test_rbd.py

```diff
@@ -29,6 +29,15 @@ def one_time_setup(node, rhbuild, branch: str) -> None:
         branch: The branch that needs to be cloned
         rhbuild: specification of rhbuild. ex: 4.3-rhel-7
     """
+    try:
+        node.exec_command(cmd="ls ceph")
+        node.exec_command(cmd="rm -rf ceph")
+        node.exec_command(
+            cmd=f"git clone --branch {branch} --single-branch --depth 1 {TEST_REPO}"
+        )
+        return
+    except BaseException:  # noqa
+        pass
```

Comment:
yes, that makes sense.. it fits nicely into the scheme of things... could you do this instead

```
node.exec_command(cmd=f"rm -rf ceph && git clone --branch {branch} --single-branch --depth 1 {TEST_REPO}")

try:
  node.exec(cmd="rpm -qa | grep epel-release")
  return
except BaseException:
  pass

os_ver = rbuild.split("-")[-1]
...
```